### PR TITLE
Use HEREDOCS in Dockerfiles

### PR DESCRIPTION
**Pull request by rahilarious**

### CODE REVIEW

File: portage.Dockerfile [hunk 1]
```diff
@@ -11,18 +15,28 @@ ARG SNAPSHOT="portage-latest.tar.xz"
...
+RUN <<-EOF
+    set -e
+
+    apk add --no-cache ca-certificates gnupg tar wget xz
+    wget -q "${DIST}/${SNAPSHOT}" "${DIST}/${SNAPSHOT}.gpgsig" "${DIST}/${SNAPSHOT}.md5sum"
+
+    # setup GPG
+    gpg --list-keys
+    # following MUST be without intendation because alpine's shell doesn't support <<-EOF syntax
+    cat <<GPG >> ~/.gnupg/dirmngr.conf
```
Comment (thesamesam) [inline portage.Dockerfile:27]: If you do `<<-GPG` here, you can indent the `honor-http-proxy`, `disable-ipv6`, and `GPG` lines.

Comment (rahilarious) [reply portage.Dockerfile:27]: We're using alpine's image here, so need to follow busybox sh's rules

Comment (thesamesam) [reply portage.Dockerfile:27]: meh, ok

Comment (rahilarious) [reply portage.Dockerfile]: It does support it. My testing method was wrong. Lmao. I was using 4 spaces instead of <tab>.

Comment (thesamesam) [reply portage.Dockerfile]: 👀

**Review by ajakk — APPROVED** [review]

File: portage.Dockerfile [hunk 2]
```diff
@@ -11,18 +15,29 @@ ARG SNAPSHOT="portage-latest.tar.xz"
...
- && gpg --verify "${SNAPSHOT}.gpgsig" "${SNAPSHOT}" \
- && md5sum -c ${SNAPSHOT}.md5sum \
- && mkdir -p var/db/repos var/cache/binpkgs var/cache/distfiles \
- && tar xJpf ${SNAPSHOT} -C var/db/repos \
- && mv var/db/repos/portage var/db/repos/gentoo \
- && rm ${SNAPSHOT} ${SNAPSHOT}.gpgsig ${SNAPSHOT}.md5sum
+RUN <<-EOF
+    set -e
+
+    apk add --no-cache ca-certificates gnupg tar wget xz
```
Comment (robbat2) [inline portage.Dockerfile:21]: tar & wget are already present in the image

**Comment by ajakk** [comment]
Thanks. I'd prefer to avoid sniping at other projects/people in commit messages, though. No need for drama. Merged with 4130317..ee2fbe

**Comment by rahilarious** [comment]
> Thanks. I'd prefer to avoid sniping at other projects/people in commit messages, though. No need for drama. Merged with [4130317](https://github.com/gentoo/gentoo-docker-images/commit/41303173bdf08e407fb2c39b173efaf6c33051df)..[ee2fbe2](https://github.com/gentoo/gentoo-docker-images/commit/ee2fbe2c8354887ff90dd908bc87964bc85f6e9f)

noted

**Pull request closed without merging**